frontend/src/app/main/data/users.cljs

```diff
@@ -450,6 +449,19 @@
         (->> (rp/query :team-users {:team-id team-id})
              (rx/map #(partial fetched %)))))))
 
+(defn fetch-file-comments-users
+  [{:keys [team-id] :as params}]
+  (us/assert ::us/uuid team-id)
+  (letfn [(fetched [users state]
+            (->> users
+                 (d/index-by :id)
+                 (assoc state :file-comments-users)))]
+    (ptk/reify ::fetch-team-users
+      ptk/WatchEvent
+      (watch [_ state _]
+        (->> (rp/query :file-comments-users {:team-id team-id :share-id (get-in state [:viewer-local :share-id])})
```

Comment:
When the map extraction is compile time known, is better use:

```clojure
(-> state :viewer-local :share-id)
```

Instead of
```clojure
(get-in state [:viewer-local :share-id])
```

Reply:
and in a separate let binding for readability

Reply:
done